ompi/mca/coll/base/coll_base_alltoall.c

```diff
@@ -472,8 +472,8 @@ int ompi_coll_base_alltoall_intra_linear_sync(const void *sbuf, int scount,
     /* find a real error code */
     if (MPI_ERR_IN_STATUS == error) {
         for( ri = 0; ri < nreqs; ri++ ) {
-            if (MPI_REQUEST_NULL == reqs[ri]) continue;
-            if (MPI_ERR_PENDING == reqs[ri]->req_status.MPI_ERROR) continue;
+            if (MPI_REQUEST_NULL == reqs[ri]) { continue; }
```

Comment:
Ewww!  This is the worst of the options.  Please don't do this.

@jsquyres is wrong.  Single line ifs without braces have always been allowed in OMPI and are found throughout the code.  However, if you really want braces, then you should follow the multi-line format:

```
if (MPI_REQUEST_NULL == reqs[ri]) {
    continue;
}
```

Given that you didn't change these lines of code, I'd abandon this entire commit and just stick to the original bug fix.

Reply:
Sure. Abandoned that commit.